File: Iterations/BinaryGap.js
function solution(N) {
    // Implement your solution here
    let count = 0;
    let max_count = 0;
    let bin = N.toString(2);

    for(let i=0; i<bin.length; i++){
        {
            if(bin[i] === '0'){
                count++;
            } else {
                max_count = Math.max(count, max_count)
                count = 0;
            }
        }
    }
    
    return max_count;
}

File: Iterations/BinaryGap.py
def solution(N):
    # Implement your solution here
    count = 0
    max_count = 0

    for i in bin(N)[2:]:
        if i == '0':
            count += 1
        else:
            max_count = max(count, max_count)
            count = 0
            
    return max_count